ios/brave-ios/Sources/Web/WebKit/WebKitPrivateHelpers.swift

```diff
@@ -15,10 +15,11 @@ extension WKWebView {
   }
 
   var dataForDisplayedPDF: Data? {
-    guard responds(to: Selector(("_dataForDisplayedPDF"))) else {
+    let selector = Selector(("_dataForDisplayedPDF"))
```

Comment:
In the future we can change this to `- (void)_getMainResourceDataWithCompletionHandler:(void (^)(NSData *, NSError *))completionHandler`